src/patternfly/components/Sidebar/sidebar.scss

```diff
@@ -152,11 +181,24 @@ $pf-c-sidebar__panel--list--width: ("default", 25, 33, 50, 66, 75, 100);
 
 .pf-c-sidebar__content {
   flex-grow: 1;
+  order: var(--pf-c-sidebar__content--Order);
   background-color: var(--pf-c-sidebar__content--BackgroundColor);
 
+  &.pf-m-padding {
+    padding: var(--pf-c-sidebar__content--PaddingTop) var(--pf-c-sidebar__content--PaddingRight) var(--pf-c-sidebar__content--PaddingBottom) var(--pf-c-sidebar__content--PaddingLeft);
+  }
+
   &.pf-m-no-background {
     --pf-c-sidebar__content--BackgroundColor: transparent;
   }
+
+  & + .pf-c-sidebar__panel {
+    --pf-c-sidebar__panel--Order: 2;
+  }
+
+  :where(&:first-child) {
```

Comment:
I wonder if it would be more flexible just to have the panel always on the left, and if you want it on the right (either by the class modifier or in the markup), you use `.pf-m-panel-right`. Looking at it again, we allow the panel on the right when the panel is first in the markup, but we don't allow the panel on the left when the panel is last in the markup. Using `.pf-m-panel-right` alone to switch the order would allow for that. wdyt?